CHANGELOG.rst

```diff
@@ -52,6 +52,7 @@ Changed
   connection related errors, our code would first wait for this timeout to be reached (30 seconds)
   before returning error to the end user. #4834
 * Upgrade ``pymongo`` to the latest stable version (``3.10.0.``). #4835 (improvement)
+* Updated Parmiko to v2.7.1 to support new key formats #4891 (improvement)
```

Comment:
```suggestion
* Updated Parmiko to v2.7.1 to support new key formats #4901 (improvement)
```
Created a suggestion out of @armab comment so all you should have to do is hit the "Accept Suggestion" button.